src/components/_General/_FormikElements/Select.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { FieldHookConfig, useFormikContext } from 'formik';
+
+import ReactSelect from 'react-select';
+import CreatableReactSelect from 'react-select/creatable';
+
+import { useReactSelectStyles } from './common';
+import FieldContainer from './FieldContainer';
+
+type SelectOption = {
+  label: string;
+  value: string;
+};
+
+interface SelectProps {
+  creatable?: boolean;
+  label?: string;
+  help?: string;
+  options: SelectOption[];
+}
+
+const Select: React.FC<SelectProps & FieldHookConfig<string>> = ({
+  creatable,
+  label,
+  help,
+  placeholder,
+  options,
+  ...props
+}) => {
+  const SelectComponent = creatable ? CreatableReactSelect : ReactSelect;
+
+  const { setFieldValue, setFieldTouched } = useFormikContext();
+  const customStyles = useReactSelectStyles();
+
+  const handleChange = (value?: SelectOption) => {
+    console.log(value, 'value');
+    setFieldTouched(props.name, true);
+    if (!value) setFieldValue(props.name, undefined);
+    else setFieldValue(props.name, value.value);
+  };
+
+  return (
+    <FieldContainer label={label} help={help} {...props}>
+      <SelectComponent
+        placeholder={placeholder}
+        onChange={handleChange}
+        options={options}
+        styles={customStyles}
+        isClearable
+      />
+    </FieldContainer>
+  );
+};
+
+Select.defaultProps = {};
```

Comment:
Probably don't need empty defaultProps in all these components. Is there a reason we need them?

Reply:
I have a small shortcut to create a basic FC component and those are there by default. Could be useful to enforce the usage of `defaultProps` but I'll remove them for now as they're just boilerplate.